doc/esp8266wifi/generic-class.rst

```diff
@@ -61,7 +61,7 @@ Other Function Calls
 .. code:: cpp
 
     int32_t  channel (void)
-    bool  setSleepMode (WiFiSleepType_t type)
+    bool  setSleepMode (WiFiSleepType_t type, int listenInterval=-1) (DTIM setting for LIGHT or MODEM sleep mode)
```

Comment:
Rather than add a short explanation here, how about adding a section above with the explanation, and adding a short comment for the args?